ext/skeleton/.gitignore.in

```diff
@@ -8,11 +9,13 @@ build
 config.guess
 config.h
 config.h.in
+*~
```

Comment:
Perhaps if we add this one on top somewhere together with those *.

Reply:
It looked sorted, so I added it to the bottom of the list.